.github/workflows/Pipeline.yml

```diff
@@ -139,3 +139,25 @@ jobs:
           check_name: Test Results
           github_token: ${{ secrets.GITHUB_TOKEN }}
           files: artifacts/**/etest.xml
+
+  #--------------------------------------------------------------------------------------------------------------------
+  #
+  # Release
+  #
+  #--------------------------------------------------------------------------------------------------------------------
+  release:
+    name: "Release"
+    needs: [Linux, MacOS]
+    runs-on: ubuntu-latest
+    if: success() && github.ref == 'refs/heads/develop'
```

Comment:
This part is important. This means we will only merge code from `develop` -> `main` if all the builds passed and we're on the `develop` branch.